docs/admin/setup/pyris.rst

```diff
@@ -31,70 +25,214 @@ E.g.: ``git clone https://github.com/ls1intum/Pyris.git Pyris``
+      api_key: "<your_azure_api_key>"
+      tools: []
+      capabilities:
+        input_cost: 6
+        output_cost: 16
+        gpt_version_equivalent: 4.5 # This is the equivalent GPT version of the model. We use 4.5 for GPT 4 Omni model.
+        context_length: 128000
+        vendor: "OpenAI"
+        privacy_compliance: True
+        self_hosted: False
+        image_recognition: True
+        json_mode: True
+      description: "GPT 4 Omni on Azure"
+      id: "azure-gpt-4-omni"
+      name: "GPT 4 Omni"
+      type: "azure_chat"
+      endpoint: <your_azure_model_endpoint>
+      api_version: "2024-02-15-preview"
+      azure_deployment: "gpt4o"
+      model: "gpt4o"
 
 Start Pyris
 """""""""""
+Using local environment
+***********************
+
+.. warning::
+    For local Weaviate vector database setup, please refer to `Weaviate Docs <https://weaviate.io/developers/weaviate/quickstart>`_.
+
+=============
+Prerequisites
+=============
+- Clone the Pyris repository to your local machine.
+- Ensure you correctly configured ``llm_config.local.yml`` file.
+- Ensure you correctly configured ``application.local.yml`` file.
+
+=======================================================
+Setup instructions
+=======================================================
+Follow the following steps for the local Pyris setup:
+
+1. **Check python version:** python --version (should be 3.12)
+2. **Install packages:** pip install -r requirements.txt
+3. **Start Pyris** using the following command:
+    .. code-block:: bash
+
+      APPLICATION_YML_PATH=<path-to-your-application-yml-file> LLM_CONFIG_PATH=<path-to-your-llm-config-yml> uvicorn app.main:app --reload
+
+4. **You can now access the API docs under the following link:** http://localhost:8000/docs
+
+This setup should help you run the Pyris application on your local machine.
+Ensure you modify the configuration files as per your specific requirements before deploying.
+
+Using Docker
+************
+
+You can run Pyris in different environments: ``development`` or ``production``. Docker Compose is used to orchestrate the different services, including ``Pyris``, ``Weaviate``, and ``Nginx``.
+
+=============
+Prerequisites
+=============
+
+-  Ensure Docker and Docker Compose are installed on your machine.
+-  Clone the Pyris repository to your local machine.
+
+=======================================================
+Setup Instructions
+=======================================================
+1. **Build and Run the Containers**
+   You can run Pyris in different environments: development or
+   production. Docker Compose is used to orchestrate the different
+   services, including Pyris, Weaviate, and Nginx.
+   -  **For Development:**
+
+      Use the following command to start the development environment:
+
+      .. code:: bash
+
+         docker-compose -f docker-compose/pyris-dev.yml up --build
```

Comment:
I think you should use the newer `docker- compose` without the `-` here

Reply:
I agree with Simon, we should `docker compose`

Reply:
I have updated the docker compose syntax. Thank you for the review.